Sources/MessagingPushAPN/rich push/RichPushRequest.swift

```diff
@@ -0,0 +1,33 @@
+import CioMessagingPush
+import Foundation
+#if canImport(UserNotifications)
+import UserNotifications
+
+internal class RichPushRequest {
+    private let completionHandler: (UNNotificationContent) -> Void
+    private let pushContent: PushContent
+
+    init(
+        pushContent: PushContent,
+        request: UNNotificationRequest,
+        completionHandler: @escaping (UNNotificationContent) -> Void
+    ) {
+        self.completionHandler = completionHandler
+        self.pushContent = pushContent
+    }
+
+    func start() {
```

Comment:
`start` is meant to mean to start the rich push request since the class is named `RichPushRequest`. 

The function doesn't do much at this time but it's plumbing for me to do images next. We start async operations when you call this function.